src/Compilers/CSharp/Portable/Symbols/ConstraintsHelper.cs

```diff
@@ -797,6 +821,52 @@ private static bool HasDuplicateInterfaces(NamedTypeSymbol type, ConsList<TypeSy
                 skipParameters);
         }
 
+        private struct ConversionsUtil
```

Comment:
>private struct ConversionsUtil [](start = 8, length = 30)

It looks like this mechanism is redundant. Does it duplicate what ```ConversionsBase._lazyOtherNullability``` is used for?

Reply:
Indeed it does

---
In reply to: [255382230](https://github.com/dotnet/roslyn/pull/33261#discussion_r255382230) [](ancestors = 255382230)